src/api/config/routes/webui_routes.rb

```diff
@@ -123,6 +123,10 @@
       resources :branches, controller: 'webui/packages/branches', only: [:create], constraints: cons do
         get '/:project', action: :into, on: :new, as: :project, constraints: cons
       end
+      resource :change_devel_projects, controller: 'webui/requests/change_devel_projects', only: [] do
```

Comment:
What is this `only: []` doing?

Reply:
with `only: []`, we will not generate the default actions 'index, edit, update,...'

Reply:
Oh! Nice! Thanks!